From 6a8446ac947f67ddbe7e1cc2c48a40adc6ca7eca Mon Sep 17 00:00:00 2001
From: Yousong Zhou <yszhou4tech@gmail.com>
Date: Tue, 11 Nov 2014 06:03:19 +0800
Subject: [PATCH 4/7] examples: fix in tree build.

---
 examples/runqueue-example.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/examples/runqueue-example.c b/examples/runqueue-example.c
index 1ae184a..16e6cf9 100644
--- a/examples/runqueue-example.c
+++ b/examples/runqueue-example.c
@@ -16,7 +16,7 @@
  * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  */
 
-#include <libubox/uloop.h>
+#include <uloop.h>
 #include <stdlib.h>
 #include <stdio.h>
 #include <unistd.h>
-- 
1.7.10.4

