From 2f84fb913ac7161328ce56a0dcc67c0b8f9840c1 Mon Sep 17 00:00:00 2001
From: Yousong Zhou <yszhou4tech@gmail.com>
Date: Tue, 11 Nov 2014 08:03:07 +0800
Subject: [PATCH 7/7] cmake: fix build with MacPorts.

---
 lua/CMakeLists.txt |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lua/CMakeLists.txt b/lua/CMakeLists.txt
index 10c6dc1..bd8164b 100644
--- a/lua/CMakeLists.txt
+++ b/lua/CMakeLists.txt
@@ -8,7 +8,7 @@ IF(NOT LUA_CFLAGS)
 	FIND_PROGRAM(PKG_CONFIG pkg-config)
 	IF(PKG_CONFIG)
 		EXECUTE_PROCESS(
-			COMMAND pkg-config --silence-errors --cflags lua5.1
+			COMMAND pkg-config --silence-errors --cflags lua-5.1
 			OUTPUT_VARIABLE LUA_CFLAGS
 			OUTPUT_STRIP_TRAILING_WHITESPACE
 		)
-- 
1.7.10.4

